isis/src/base/apps/phocube/tsts/off_body/Makefile

```diff
@@ -0,0 +1,10 @@
+# Tests RA and declination bands to make sure values are displayed even if not on the planet.
+APPNAME = phocube
+
+include $(ISISROOT)/make/isismake.tsts
+
+commands:
+	 $(APPNAME) from=$(INPUT)/EW0131773041G.cal.cub \
```

Comment:
Would it be useful to also test the cube label for this test?

Reply:
Would we be checking the label to see if the BandBin group is ordered correctly or is there another reason why we should? I think this is a good idea either way.

Reply:
I believe that the app test cubediff only checks some of the label. Generally, if you want to check things in the cube label, you should use catlab to output it as a PVL.

Reply:
`cubediff` does not check the cube label at all -- just the DNs. :+1: on needing to catlab the cube label out to a pvl to check the label.